### ISSUE TITLE
trivial documentation issue (non escaped html escapes in docs of `escape_htm`l)

### ISSUE THREAD

**Issue by dathinab** [Keats/tera#192]
The doc of [`escape_html`](https://github.com/Keats/tera/blob/master/src/utils.rs#L9-L14)
has two small problems:

1. the list of "how it's escaped" mappings is inlined as there is no blank line between them (neither is a markdown list used) 

2. the escapes are not escape (`&gt;` is displayed as `>` in the docs)

e.g. change
```
& --> &amp;
< --> &lt;
> --> &gt;
[...]
```
to
```
- `&` => `&amp`
- `<` => `&lt;`
- `>` => `&gt;`
[...]
```

also changes `-->` to `=>` for better readability as `-->` also happens to be
the end of a HTML comment tag

(will look like this:
- `&` => `&amp`
- `<` => `&lt;`
- `>` => `&gt;`
[...]

)

**Comment by Keats** [comment]
Thanks for the report, I'd love a PR if you have time!

**Comment by Keats** [comment]
That was fixed in #200 and will be in the next release!
